examples/supabase.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @title Connect to Supabase
+ * @difficulty intermediate
+ * @tags cli, deploy
+ * @run --allow-net --allow-env <url>
+ * @resource {https://supabase.com/docs/reference/javascript/installing} supabase-js docs
+ * @resource {https://jsr.io/@supabase/supabase-js} supabase-js on jsr.io
+ * @group Databases
+ *
+  * Connect to a Supabase database with the supabase-js library.
+ */
+
```

Comment:
/**
 * @title Connect to Supabase
 * @difficulty intermediate
 * @tags cli, deploy
 * @run --allow-net --allow-env <url>
 * @resource {https://supabase.com/docs/reference/javascript/installing} supabase-js docs
 * @resource {https://jsr.io/@supabase/supabase-js} supabase-js on jsr.io
 * @group Databases
 *
 * Connect to a Supabase database with the supabase-js library.
 */